scripts/update-versions-for-release

```diff
@@ -56,11 +57,11 @@ update_versions_in_chart() {
 
     sed -i '' "s/version: $old_chart_version/version: $new_chart_version/g" $CHART
     sed -i '' "s/amazon-ec2-metadata-mock-$old_chart_version.tgz/amazon-ec2-metadata-mock-$new_chart_version.tgz/g" $CHART_README
-    echo -e "\nUpdated chart version from $old_chart_version to $new_chart_version in files: \n$CHART\n$CHART_README"
+    echo -e "\n✅ Updated chart version from $old_chart_version to $new_chart_version in files: \n$CHART\n$CHART_README"
```

Comment:
just fyi: In a PR I'm working on, I think we should remove the chart version from the **.tgz** and therefore the readme as well. Reasoning is the download link to the .tgz already contains the release version as an identifier for which charts a user downloads.

Reply:
Helm charts usually come with chart versions - I think this adds clarity, before, during and after downloading a chart archive. See [this](https://helm.sh/docs/topics/charts/#charts-and-versioning) for details. 

And updating the chart version is already automated, and is done along with other version updates, so shouldn't be problematic.